README.md

```diff
@@ -2,7 +2,7 @@
 [![codecov](https://codecov.io/gh/material-components/material-components-web/branch/master/graph/badge.svg)](https://codecov.io/gh/material-components/material-components-web)
 [![Chat](https://img.shields.io/discord/259087343246508035.svg)](https://discord.gg/material-components)
 
-# Material Components for the web
+# Material Components for the Web
```

Comment:
You said you're making the headings more consistent, but the capitalization of web here is inconsistent with the use of "for the web" elsewhere.

Reply:
Reverted